Solutions/CohesitySecurity/Package/createUiDefinition.json

```diff
@@ -60,7 +60,17 @@
             "name": "dataconnectors1-text",
             "type": "Microsoft.Common.TextBlock",
             "options": {
-              "text": "This product integrates Cohesity Helios with Microsoft Sentinel to stay updated with the security events from your Cohesity environment and immediately respond to a ransomware attack or an anomaly."
+              "text": "This Solution installs the data connector for CohesitySecurity. You can get CohesitySecurity custom log data in your Microsoft Sentinel workspace. Configure and enable this data connector in the Data Connector gallery after this Solution deploys. This data connector creates custom log table(s) Cohesity_CL in your Microsoft Sentinel / Azure Log Analytics workspace."
```

Comment:
The new text is incorrect. We don't have data connectors and custom tables.
I'd leave the previous text
`This product integrates Cohesity Helios with Microsoft Sentinel to stay updated with the security events from your Cohesity environment and immediately respond to a ransomware attack or an anomaly.`

Reply:
@kishannerella manually change this from the generated json file.
this is incorrect, I need to fix it from the build script.

Reply:
maybe I could also spend some time to fix the email id issue from the build script.